packages/model-viewer/src/features/scene-graph/material.ts

```diff
@@ -123,4 +135,55 @@ export class Material extends ThreeDOMElement implements MaterialInterface {
     (this[$sourceObject] as GLTFMaterial).emissiveFactor = rgb;
     this[$onUpdate]();
   }
+
+  setAlphaCutoff(value: number) {
+    (this[$sourceObject] as GLTFMaterial)
+    for (const material of this[$correlatedObjects] as
+         Set<MeshStandardMaterial>) {
+      // This little hack ignores alpha for opaque materials, in order to comply
+      // with the glTF spec.
+      if (!material.alphaTest && !material.transparent) {
+        material.alphaTest = -0.5;
+      } else if (this.alphaMode === 'MASK') {
+        material.alphaTest = value;
```

Comment:
It might be nice to make functions of things like this so that we can reduce duplication between here and `ModelViewerGLTFInstance.ts`.